digits/model/tasks/caffe_train.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, **kwargs):
         self.solver = None
 
         self.solver_file = CAFFE_SOLVER_FILE
-        self.original_file = CAFFE_ORIGINAL_FILE
+        self.model_file = CAFFE_ORIGINAL_FILE
```

Comment:
Can we use `network_file` instead of `model_file` here? I try to be consistent in calling the .prototxt file a "network description" and not calling it a model unless it has weights attached to it.

The nomenclature I'd like to migrate to (but don't fully support yet) is:

* "network" for a .prototxt file
* "model" or "trained model" for a .prototxt and a corresponding .caffemodel file
* "training" for a group of models